tests/queries/0_stateless/02174_cte_scalar_cache.sql

```diff
@@ -46,3 +56,17 @@ WHERE
   AND type = 'QueryFinish'
   AND query LIKE '%02177_CTE_GLOBAL_OFF%'
   AND event_date >= yesterday() AND event_time > now() - interval 10 minute;
+
+SELECT
```

Comment:
Need to add test specifically for analyzer scalar subqueries.

Reply:
This test with '02177_CTE_NEW_ANALYZER', is for analyser scalar subqueries with `SELECT sleep(0.0001) FROM system.one` being the scalar subquery. 
If you have any other queries which should be added to the test, could you please share some examples.